Header header
Header image_header
int64[] image_count
int64[] vehicle_count
int64[] roi_count
